src/app/layout/footer/footer.component.html

```diff
@@ -1,19 +1,41 @@
 <footer class="page-footer">
   <div class="container">
-    <div class="row">
+    <div class="row" data-aos="fade-up" data-aos-anchor-placement="top-bottom" data-aos-duration="1000">
       <div class="col l6 s12">
-        <h5 class="white-text">Pie de página</h5>
-        <p class="grey-text text-lighten-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-          tempor incididunt ut labore et dolore magna aliqua.</p>
+        <div id="footer-logo"></div>
+        <div class="row">
+          <h5 class="white-text">Información de contacto</h5>
+        </div>
+        <div class="row">
+          <i class="fas fa-mobile-alt fa-1x fa-inverse"></i>
+          <span class="grey-text text-lighten-4"> Celular: (+591) 78364554.</span>
+        </div>
+        <div class="row">
+          <i class="fas fa-envelope-square fa-1x fa-inverse"></i>
+          <span class="grey-text text-lighten-4"> Correo:
+            <a class="grey-text text-lighten-4" id="mailto-link"
+              href="mailto:ba.lamanzanaroja.bolivia@gmail.com">ba.lamanzanaroja.bolivia@gmail.com</a>
+          </span>
+        </div>
+        <div class="row">
+          <i class="fas fa-map-marker-alt fa-1x fa-inverse"></i>
+          <span class="grey-text text-lighten-4"> Ubicación: Avenida Ayacucho Nro. 56</span>
+        </div>
       </div>
       <div class="col l4 offset-l2 s12">
-        <h5 class="white-text">Enlaces</h5>
-        <ul>
-          <li><a class="grey-text text-lighten-3">Enlace 1</a></li>
-          <li><a class="grey-text text-lighten-3">Enlace 2</a></li>
-          <li><a class="grey-text text-lighten-3">Enlace 3</a></li>
-          <li><a class="grey-text text-lighten-3">Enlace 4</a></li>
-        </ul>
+        <div class="row">
+          <h5 class="white-text">Redes sociales</h5>
+        </div>
+        <div class="row">
+          <div class="col l2 s3">
+            <a class="grey-text text-lighten-4" href="https://www.facebook.com/LaManzanaRojaBolivia" target="_blank"><i
+                class="fab fa-facebook fa-2x"></i></a>
+          </div>
+          <div class="col l2 s3">
+            <a class="grey-text text-lighten-4" href="https://www.instagram.com/lamanzanaroja.ba" target="_blank"><i
```

Comment:
Please add rel="noopener noreferrer" to elements that open content in a blank tab, this is due to security and performance issues: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target